### ISSUE TITLE
Status?

### ISSUE THREAD

**Issue by pepoospina** [optimism-java/thrud#3]
Hi there, is this project active? I'm building a sample wallet app that use ERC4337 and would be building UserOperation stuff there. I'm not a kotlin expert, so if you know of any existing code I could use as reference, that would be super useful!

Thanks
Pepo

**Comment by thinkAfCod** [comment]
The project has just begun and may take one to two months to complete.

**Comment by pepoospina** [comment]
Are you going to use https://github.com/komputing/KEthereum?

**Comment by thinkAfCod** [comment]
No, we will make reference to the project through the link provided below:
https://www.erc4337.io/bundlers
May find something you need, I guess?

**Comment by pepoospina** [comment]
I didn't see anything in Kotlin in that link, but thanks.

I'm looking for the "ethers.js but for Kotlin" library. A library to handle typical EVM operations like RPC calls to a node, signers/providers management, etc, to do this from Android.

Are you planing to code this from scratch or use some library?

**Comment by thinkAfCod** [comment]
Like this https://github.com/tmio/tuweni?
But it is not implements ERC4337.

**Comment by saugion** [comment]
@pepoospina maybe web3j? i have a full project based on it, send a useroperation is possible, you can copy the signature management from here: https://medium.com/web3labs/meta-transactions-using-web3j-7c9a358db5b6

**Comment by pepoospina** [comment]
Thanks I will explore these.
